sentry-samples/sentry-samples-android/src/main/java/io/sentry/samples/android/MyApplication.java

```diff
@@ -16,12 +13,17 @@ public void onCreate() {
 
     // Example how to initialize the SDK manually which allows access to SentryOptions callbacks.
     // Make sure you disable the auto init via manifest meta-data: io.sentry.auto-init=false
-    SentryAndroid.init(
-        this,
-        options -> {
-          options.addIntegration(new FragmentLifecycleIntegration(MyApplication.this, true, true));
-          options.addIntegration(new SentryTimberIntegration());
-        });
+    // SentryAndroid.init(
```

Comment:
Is this supposed to be uncommented or deleted before we merge?

Reply:
This was a suggestion from Manoel https://github.com/getsentry/sentry-java/pull/1936#discussion_r820670053, this way we could show those who are looking at the sample code that it's still possible to manually init the sdk

Reply:
Sample code can lode outside of production code. I think this will end up becoming a source of maintenance

Reply:
that is also true for the commented manifest entries now. I think I'd keep it like that, find+replace on text should reveal this code as well :)